python/ray/serve/api.py

```diff
@@ -1802,6 +1831,35 @@ def run(
     client.deploy_group(parameter_group, _blocking=True)
     return deployments[-1].get_handle()
 
+    # TODO (shrekris-anyscale): return handle to ingress deployment
+
+
+def deploy_group(deployments: List[Deployment], *, blocking=True) -> RayServeHandle:
+    """Atomically deploys a group of deployments."""
+
+    if len(deployments) == 0:
+        return
+
+    parameter_group = []
+
+    for deployment in deployments:
+        deployment_parameters = {
+            "name": deployment._name,
+            "func_or_class": deployment._func_or_class,
+            "init_args": deployment.init_args,
+            "init_kwargs": deployment.init_kwargs,
+            "ray_actor_options": deployment._ray_actor_options,
+            "config": deployment._config,
+            "version": deployment._version,
+            "prev_version": deployment._prev_version,
+            "route_prefix": deployment.route_prefix,
+            "url": deployment.url,
+        }
+
+        parameter_group.append(deployment_parameters)
+
+    internal_get_global_client().deploy_group(parameter_group, _blocking=blocking)
```

Comment:
does `deploy_group` return a `RayServeHandle` as func signature says ?

Reply:
Ah, not it doesn't. However, it also doesn't look like anything is using the public `deploy_group`, so I'll remove it.